doc/user/content/transform-data/json.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "JSON to SQL"
+description: "Create a view from sample JSON data."
+menu:
+  main:
+    parent: 'transform'
+    name: JSON
+    weight: 20
+---
+
+Working with JSON formatted data in SQL can be tedious, because you need to constantly deconstruct
+the JSON blob. The below tool accepts a JSON sample and will generate a SQL view with the
+individual fields mapped to columns.
+
+
+<div class="json_widget">
+
+<span class="input_container">
+
+<span class="input_container-text">
+<input id="view_name" placeholder="View Name">
```

Comment:
This would be easier to read if we indented it, I think Prettier would do this? Not sure if there is already a formatting tool enforced for markdown though.

Reply:
When I first tried indenting Hugo interpreted it as plain text, which is why I left it flat. Just tried again without line breaks between elements and it seems to have worked!